test/Microsoft.DotNet.Tools.Tests.Utilities/Commands/TestCommand.cs

```diff
@@ -33,6 +33,12 @@ public TestCommand(string command)
 #endif 
         }
 
+        public TestCommand WithWorkingDirectory(string workingDirectory)
+        {
+            WorkingDirectory = workingDirectory;
+            return this;
```

Comment:
Last one in needs to merge.  I won't overwrite a blank line.

BTW - I have the same method here:

https://github.com/dotnet/cli/pull/2700/files#diff-5904959b36713999737967faf1da4ccfR38

So I'll conflict with myself.